src/core_atmosphere/Registry.xml

```diff
@@ -101,7 +101,7 @@
 
                 <nml_option name="config_dynamics_split_steps" type="integer" default_value="3"
                      units="-"
-                     description="When config\_split\_dynamics\_transport = T, the number of RK steps per transport step"
+                     description="When config_split_dynamics_transport = T, the number of RK steps per transport step"
                      possible_values="Positive integer values"/>
```

Comment:
@mgduda I assume this is a no-impact change...

Reply:
Correct. There are actually two commits in this PR. The second commit simply fixes some documentation strings to match what we need when creating LaTeX source for the users' guide. This is mentioned in the commit message for the second commit.